terraform/key-vault.tf

```diff
@@ -0,0 +1,29 @@
+data "azurerm_client_config" "current" {}
+
+variable "msi_id" {
```

Comment:
I think these should be moved to the proper local/variable files.  I can do this when committing the changes for the managed identity stuff.

Reply:
Ok I'll leave the move of the variable to the lcoals.tf to you in your merge